pycbc/io/live.py

```diff
@@ -166,30 +170,16 @@ def __init__(self, ifos, coinc_results, **kwargs):
             if self.snr_series is not None:
                 snr_series_to_xml(self.snr_series[ifo], outdoc, sngl.event_id)
 
-        # Set merger time to the average of the ifo peaks
-        self.merger_time = numpy.mean(
-                    [coinc_results[f'foreground/{ifo}/end_time']
-                     for ifo in ifos]) + self.time_offset
-
-        # For subthreshold detectors, respect BAYESTAR's assumptions and checks
-        bayestar_check_fields = ('mass1 mass2 mtotal mchirp eta spin1x '
-                                 'spin1y spin1z spin2x spin2y spin2z').split()
-        for sngl in sngl_inspiral_table:
-            if sngl.ifo in followup_ifos:
-                for bcf in bayestar_check_fields:
-                    setattr(sngl, bcf, getattr(sngl_populated, bcf))
-                sngl.end = lal.LIGOTimeGPS(self.merger_time)
```

Comment:
Are you intending to remove this? I do not see the same functionality being reimplemented elsewhere.

Reply:
Yes, because with the O3 behaviour followup_ifos was by design an empty list and all the sngl inspiral fields are populated in the SNR followup code.

Reply:
Yeah you are right that this code had no effect when the class is used by PyCBC Live. I am not sure about `pycbc_make_skymap` though, that should be checked. We may also need this code again once we implement "KAGRA mode", depending on how that is implemented.

Reply:
I would be OK with not removing this if the set of ifos it potentially concerns can be clearer. 'Followup' could refer either to significance followup or EM followup ie sky map.
In either case the ifo requires a SNR time series to be calculated, though, and I don't know where that would happen that doesn't already populate this information elsewhere (?)